### ISSUE TITLE
No such file

### ISSUE THREAD

**Issue by Johnsonzcode** [TheJacksonLaboratory/nanome#131]
```
(nanome) [poultrylab1@pbsnode01 nanome]$ nextflow run TheJacksonLaboratory/nanome -profile test,docker
N E X T F L O W  ~  version 21.10.0
Launching `TheJacksonLaboratory/nanome` [chaotic_hamilton] - revision: c181f907e9 [master]
NANOME - NF PIPELINE (v1.3.6)
by Li Lab at The Jackson Laboratory
https://github.com/TheJacksonLaboratory/nanome
=================================
dsname              : CIEcoli
input               : https://github.com/TheJacksonLaboratory/nanome/raw/master/test_data/ecoli_ci_test_fast5.tar.gz
genome              : ecoli

Running settings   : --------
processors          : 2
chrSet              : NC_000913.3
dataType            : ecoli
runBasecall         : Yes
runNanopolish       : Yes
runMegalodon        : Yes
runDeepSignal       : Yes
runGuppy            : Yes

Pipeline settings  : --------
Working dir         : /storage-04/chicken/ont_methylation/nanome/work
Output dir          : outputs
Launch dir          : /storage-04/chicken/ont_methylation/nanome
Script dir          : /storage-01/poultrylab1/.nextflow/assets/TheJacksonLaboratory/nanome
User                : poultrylab1
Profile             : test,docker
Config Files        : /storage-01/poultrylab1/.nextflow/assets/TheJacksonLaboratory/nanome/nextflow.config
Pipeline Release    : master
Container           : docker - liuyangzzu/nanome:v1.2
=================================
executor >  local (1)
[6e/af606b] process > EnvCheck (EnvCheck) [100%] 1 of 1 ✔
[-        ] process > Untar               -
[-        ] process > Basecall            -
[-        ] process > QCExport            -
[-        ] process > Resquiggle          -
[-        ] process > Nanopolish          -
[-        ] process > NplshComb           -
[-        ] process > Megalodon           -
[-        ] process > MgldnComb           -
[-        ] process > DeepSignal          -
[-        ] process > DpSigComb           -
[-        ] process > Guppy               -
[-        ] process > GuppyComb           -
[-        ] process > Report              -
No such file: https://github.com/TheJacksonLaboratory/nanome/raw/master/test_data/ecoli_ci_test_fast5.tar.gz


```

**Comment by liuyangzzu** [comment]
This error is very strange, since the input file link is already located at the Github. Would you please rerun to see if it is reproducible, and post the Nextflow running log/error file named `.nextflow.log`? Thank you!

**Comment by Johnsonzcode** [comment]
Yes, I checked the link `https://github.com/TheJacksonLaboratory/nanome/raw/master/test_data/ecoli_ci_test_fast5.tar.gz`, the `ecoli_ci_test_fast5.tar.gz` exists.
I realized it may be my internet connection, you know we have Inconveniences in touch `github.com`

**Comment by Johnsonzcode** [comment]
OK, I am trying to solve it as your instruction. Thank you for your quick reply !

**Comment by liuyangzzu** [comment]
Not at all. Let me know how it goes.

**Comment by Johnsonzcode** [comment]
It works fine, thank you so much !
By the way, I am curious about whether NANOME or METEORE can detect 5mC in chicken ? I know some model are trained in human.
And If it works good in chicken, some of the top four performers rely on GPU, or they may are slow. Should choose some of them if it is enough to detect 5mC?

**Comment by liuyangzzu** [comment]
For all 5mC methylation calling tools, there is no restrictions to apply them to any species. Now, I am working on opening any reference genomes for NANOME pipeline at one branch, and it will be tested and merged by this week. (ref: https://github.com/TheJacksonLaboratory/nanome/issues/130)

I recommend you run jobs using GPU, since basecalling, and the top performers e.g. Megalodon, etc. are deep learning models, that will run very fast in GPU, but will be very slow on CPU machine.

**Comment by Johnsonzcode** [comment]
By now, I can't apply NANOME for my species, right ?
Because we don't have GPU server. What is the time consuming for CPU macheine.

Thank you for your great work in advance.

**Comment by liuyangzzu** [comment]
You can try your species on the branch `enhance5`. Documentation: https://github.com/TheJacksonLaboratory/nanome/blob/enhance5/docs/Usage.md#3-support-for-other-reference-genome

Since the branch is not fully tested, I do not merge it to master, so please use below example for your species and reference genome.
Example:
```
nextflow pull TheJacksonLaboratory/nanome -r enhance5

nextflow run TheJacksonLaboratory/nanome -r enhance5\
    -profile singularity \
    --input [input-file]\
    --genome [reference-genome-dir]\
    --chrSet '[chomosomes sperated by a space]'
```

**Comment by Johnsonzcode** [comment]
Thank you, after reading paper [DNA methylation-calling tools for Oxford Nanopore sequencing: a survey and human epigenome-wide evaluation](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02510-z), 
if i have no availability in GPU server, maybe I can use CPU-fast-software right?

**Comment by Johnsonzcode** [comment]
If my basecalling was done, how can i skip this step and provide fastq for the enhance5 pipline ?

**Comment by liuyangzzu** [comment]
If there is no GPU resources, the GPU supported tasks (Megalodon, etc.)  may be slow, but you can also try to run any of them without problem. Nanopolish is the tool that not need GPU and run fast.

Currently, NANOME pipeline not support skip basecalling step, I will add this function to the brach in recent and let you know.

**Comment by Johnsonzcode** [comment]
Thank you so much, I set `runBasecall` to false, it do not surpport indeed.

**Comment by liuyangzzu** [comment]
Hi, I added the basecalled input for NANOME, please check https://github.com/TheJacksonLaboratory/nanome/blob/master/docs/SpecificUsage.md. Let me know if you have any issues.

Best

**Comment by Johnsonzcode** [comment]
> 

Thanks a lot. I will try !!!

**Comment by Johnsonzcode** [comment]
> Hi, I added the basecalled input for NANOME, please check https://github.com/TheJacksonLaboratory/nanome/blob/master/docs/SpecificUsage.md. Let me know if you have any issues.
> 
> Best

If my bascalled input has sevaral run, namely  I had sevaral input folders. How I handle them. Merge them into one folder ? Or ?

**Comment by liuyangzzu** [comment]
NANOME support running multiple jobs on each folder, there are two options: 

(1) put all folder paths in a file, the file name is suffixed with `.filelist.txt`, like: https://github.com/TheJacksonLaboratory/nanome/blob/master/inputs/na12878_chr22.filelist.txt

(2) use wildcard path matching, such as `--input  'all_data_folders/*'`, note the single quote in wildcard string is needed.

The methylation calling will be automatically invoked on each folder in parallel.
